src/cryptoadvance/specter/cli/cli_server.py

```diff
@@ -222,7 +222,11 @@ def run(debug=debug):
                       Starting in production mode instead."
                 )
                 debug = False
-            if tor or os.getenv("CONNECT_TOR") == "True":
+            if (
+                tor
+                or os.getenv("CONNECT_TOR") == "True"
+                or app.specter.config["tor_status"] == True
+            ):
```

Comment:
If I use the --tor flag there can be a situation where I then turn the tor service off from the app, then the toggle happens incorrectly...
Could add here a check that if the toggle is false then toggle here on start

Reply:
Hmm makes sense. Will update shortly..